### PR TITLE
Updated css for save query dialog box

### LINKED ISSUE #551: [STORY] UI : Update CSS Style for Save Query Dialog Box
# Description
Update the CSS style for the Save Query dialog box to ensure consistency.
Input box and the button height should be - 32px
Heading - "Save Query" - 18px
image

# Steps to reproduce
1. On Logs screen, click the "disk" icon to open Save Query dialog box.

### PR DESCRIPTION
**Pull request by PratikshaPatil77**
# Description
Updated the CSS style for the Save Query dialog box.
Input box and the button height  - 32px
Heading - "Save Query" - 18px 

Fixes #551 

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by AndrewHess — COMMENTED** [review]
You should remove the .fasthttp.gz files. We have a PR that will add that to the gitignore, so after that's merged you shouldn't be getting these extra files in your PRs

**Review by AndrewHess — COMMENTED** [review]
This looks good to me, but I'll wait for one of the frontend people to review it

**Review by sonamgupta21 — APPROVED** [review]

**Pull request merged**
